extensions/tureng/src/index.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  randomId,
+} from "@raycast/api";
+import { useState, useEffect, useRef } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+/**
+ * Translates Turkish-English bidirectionally
+ * https://tureng-api.vercel.app used as API, provided by Mehmet Burak Erman.
+ * Icon by Freepik https://www.flaticon.com/premium-icon/dictionary_3285819
+ * API-Example used: https://github.com/raycast/extensions/blob/main/examples/api-examples/src/typeahead-search.tsx
+ * @author: myucesan
+ */
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List
+      isLoading={state.isLoading}
+      onSearchTextChange={search}
+      searchBarPlaceholder="Search Turkish or English word"
+      throttle
+    >
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.word}
+      accessoryTitle={searchResult.meaning}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <OpenInBrowserAction title="Open in Browser" url={searchResult.url} />
+          </ActionPanel.Section>
+        </ActionPanel>
+      }
+    />
+  );
+}
+
+function useSearch() {
+  const [state, setState] = useState<SearchState>({ results: [], isLoading: false });
```

Comment:
```suggestion
  const [state, setState] = useState<SearchState>({ results: [], isLoading: true });
```
Fixing last part of the [empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)